api.md

```diff
@@ -10,6 +10,7 @@
 > - [chunk_relation_size](#chunk_relation_size)
 > - [chunk_relation_size_pretty](#chunk_relation_size_pretty)
 > - [create_hypertable](#create_hypertable)
+> - [CREATE INDEX](#create_index)
```

Comment:
I think we are using a different syntax for these links. `[create index (transaction per chunk)](#create_index)`. See @gayyappan 's API PR.